.ci/update-github.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -e
+
+checkForVariable "GITHUB_TOKEN"
```

Comment:
Did you run this ?
You forgot source of util.sh

Reply:
Did, Maybe I unintentionally removed it while pushing.  Done Changes